### ISSUE TITLE
[tutorial] flaskr's pytest does not pass due to test_db.py#test_get_close_db fail.

### ISSUE THREAD

**Issue by nimzo6689** [pallets/flask#3277]
tests/test_db.py#test_get_close_db does not pass.

### Actual Behavior

![image](https://user-images.githubusercontent.com/14356595/60437543-8df99300-9c49-11e9-9bed-7fed49cfcd3d.png)

### Environment

* Python version: 3.7.3
* Flask version: 1.0.3
* Werkzeug version: 0.15.4
* pytest: 5.0.0

**Comment by asottile** [comment]
I have a fix for this in #3278

**Comment by nimzo6689** [comment]
@asottile 
Great.
Closed this issue.

**Comment by safurynp** [comment]
I think this can be fixed by using the getrepr() method.

`def test_get_close_db(app):
    with app.app_context():
        db = get_db()
        assert db is get_db()

    with pytest.raises(sqlite3.ProgrammingError) as e:
        db.execute('SELECT 1')

    assert 'closed' in str(e.getrepr())`

**Comment by safurynp** [comment]
I think this can be fixed using the getrepr() method.
```python
def test_get_close_db(app):
    with app.app_context():
        db = get_db()
        assert db is get_db()

    with pytest.raises(sqlite3.ProgrammingError) as e:
        db.execute('SELECT 1')

    assert 'closed' in str(e.getrepr())
```

**Comment by asottile** [comment]
here's how it was fixed in #3278: https://github.com/pallets/flask/pull/3278/files#diff-27ab2245a7b45831d4fa8f8b51241d80R191
